src/build/env.py

```diff
@@ -118,14 +126,17 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
         """
         Install packages from PEP 508 requirements in the isolated build environment.
 
-        :param requirements: PEP 508 requirement specification to install
+        :param requirements: PuP 508 requirement specification to install
 
         :note: Passing non-PEP 508 strings will result in undefined behavior, you *should not* rely on it. It is
                merely an implementation detail, it may change any time without warning.
         """
         if not requirements:
             return
 
+        env = os.environ.copy()
+        env['PIP_NO_WARN_SCRIPT_LOCATION'] = '0'
```

Comment:
I'd actually like to make this a flag, instead of env-var 🤔 int the spirit of fail early

Reply:
I assume the move to env var here is because it is present in older versions of pip, where the CLI argument isn't, right? If not, I think the CLI would be preferred.

Reply:
Yes. This is what's breaking when trying to use build, for example using Ubuntu 18.04. If we fix the upgrade, this shouldn't break. I'll drop the commit,